src/nvim/normal.c

```diff
@@ -2145,23 +2147,39 @@ void do_check_scrollbind(bool check)
   old_leftcol = curwin->w_leftcol;
 }
 
+static int get_scrollbind_amount(win_T *orig_wp, int old_topline, int topfill_diff)
+{
+  int orig_topline = orig_wp->w_topline;
```

Comment:
The `orig_topline` name sounds confusing as it's not clear that it's newer than `old_topline`.